src/components/GlobalPreferences/Notifications/SubscriptionsForm.js

```diff
@@ -209,8 +210,17 @@ export default function SubscriptionsForm({
     )
   }
 
-  const appNames = subscribableApps.map(
-    app => `${app.name} ${app.identifier ? `(${app.identifier})` : ''}`
+  const appBadges = subscribableApps.map(
+    ({ contractAddress, icons: [{ src: iconSrc }], name, appName }) => {
+      return (
+        <AppBadge
+          appBadge={contractAddress}
+          badgeOnly
+          iconSrc={iconSrc}
```

Comment:
@sohkai do I've to use a combination of `app.baseUrl` + `iconSrc`?

Reply:
I think so, the `iconSrc` is only relative to that app's IPFS content (which needs the `baseUrl`). `AppBadge` just takes a normal URL and tries to fetch its image.

Reply:
Done. Icons render correctly now:
![image](https://user-images.githubusercontent.com/3072458/64537107-1b282a80-d312-11e9-9f08-635939697e1c.png)